bin/keycloak-httpd-client-install

```diff
@@ -1033,10 +1064,21 @@ def main():
                        help='Add SAML OrganizationURL to SP metadata')
 
     group.add_argument('-l', '--mellon-protected-locations', action='append',
-                       type=arg_type_mellon_protected_location, default=[],
-                       help='Web location to protect with Mellon. '
+                       type=arg_type_module_protected_location, default=[],
+                       help='Deprecated. Use --protected-locations instead '
+                       ' Web location to protect with Mellon. '
                             'May be specified multiple times')
 
+    # ---- Argument Group "OIDC SP" ----
+
+    # both mellon and oidc use this but idk if they should be seperate or not
+    group = parser.add_argument_group('OIDC RP')
+
+    group.add_argument('--ssl-validate-server', default='Off')
```

Comment:
I presume syntactically there are only a few valid values (e.g. On, Off). Therefore don't allow someone to set a value that can't be parsed. Using a choice argument here would be appropriate to prevent generating a syntactically invalid file.